package.json

```diff
@@ -40,6 +40,8 @@
     "babelify": "^7.2.0",
     "bluebird": "^3.0.6",
     "browserify": "^12.0.1",
+    "lodash": "^3.10.1",
+    "tsify": "^0.13.1",
```

Comment:
Ah that's probably result of the different module roots spread over the project causing dependency lookup issues. That's one of the main reasons why I'm voting for centralized dependency management. See https://github.com/serverless/serverless/issues/428.